src/styles/stars.css

```
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+}
+
+.stars--1 > :nth-child(-n+1),
+.stars--2 > :nth-child(-n+2),
+.stars--3 > :nth-child(-n+3),
+.stars--4 > :nth-child(-n+4),
+.stars--5 > :nth-child(-n+5) {
+  background: url("../images/star-active.svg") no-repeat center;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background: url("../images/star.svg") no-repeat center;
```

тут (або в 10 рядку) можна замінити на `background-image` і не прописувати другий раз `no-repeat center`
```suggestion
  background-image: url("../images/star.svg");
```

---

Тоді збивається зображення. Вернув назад.